src/recording/recordingmanager.cpp

```diff
@@ -100,17 +112,25 @@ void RecordingManager::startRecording(bool generateFileName) {
         m_recordingLocation = m_recording_base_file + "."+ encodingType.toLower();
         m_pConfig->set(ConfigKey(RECORDING_PREF_KEY, "Path"), m_recordingLocation);
         m_pConfig->set(ConfigKey(RECORDING_PREF_KEY, "CuePath"), m_recording_base_file +".cue");
+
+        m_recReady->set(RECORD_READY);
+
     } else {
         // This is only executed if filesplit occurs.
```

Comment:
How about putting the else branch into a new function like 
splitContinueRecording()?